services/src/main/java/io/druid/cli/CoordinatorJettyServerInitializer.java

```diff
@@ -59,10 +63,19 @@ public void initialize(Server server, Injector injector)
 
     root.addServlet(holderPwd, "/");
     if(config.getConsoleStatic() == null) {
-      ResourceCollection staticResources = new ResourceCollection(
-          Resource.newClassPathResource("io/druid/console"),
-          Resource.newClassPathResource("static")
-      );
+      ResourceCollection staticResources;
+      if (beOverlord) {
```

Comment:
yes, i tested this.  with this flag,

coordinator console - http://host:port/
overlord console - http://host:port/console.html